.travis.yml

```diff
@@ -204,7 +204,7 @@ matrix:
     - os: linux
       env:
         # These components will build as Habitat packages in the provided order
-        - COMPONENTS="hab-butterfly hab plan-build backline bintray-publish studio pkg-aci pkg-dockerize pkg-cfize pkg-export-docker pkg-mesosize pkg-tarize sup"
+        - COMPONENTS="hab-butterfly hab plan-build backline bintray-publish studio pkg-aci pkg-dockerize pkg-cfize pkg-export-docker pkg-export-kubernetes pkg-mesosize pkg-tarize sup"
         - AFFECTED_DIRS="\.bldr\.toml|support/ci/deploy\.sh|Cargo\.toml|Cargo\.lock|VERSION|components/backline|components/bintray-publish|components/pkg-aci|components/pkg-cfize|components/pkg-dockerize|components/pkg-mesosize|components/pkg-tarize|components/plan-build|components/studio|$_RUST_HAB_BIN_COMPONENTS|$_RUST_HAB_LIB_COMPONENTS"
```

Comment:
add the directory here so that changes triger a packageing

Reply:
@mwrock I think this is satisfied by adding it to `_RUST_HAB_BIN_COMPONENTS`